registration/src/main/java/org/spine3/samples/lobby/registration/util/MessageFactory.java

```diff
@@ -0,0 +1,140 @@
+/*
+ * Copyright 2015, TeamDev Ltd. All rights reserved.
+ *
+ * Redistribution and use in source and/or binary forms, with or without
+ * modification, must retain the above copyright notice and the following
+ * disclaimer.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS
+ * "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT
+ * LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR
+ * A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
+ * OWNER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL,
+ * SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
+ * LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,
+ * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
+ * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
+ * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+package org.spine3.samples.lobby.registration.util;
+
+import com.google.common.collect.ImmutableList;
+import com.google.protobuf.Message;
+import org.spine3.money.Currency;
+import org.spine3.money.Money;
+import org.spine3.samples.lobby.common.ConferenceId;
+import org.spine3.samples.lobby.common.OrderId;
+import org.spine3.samples.lobby.common.ReservationId;
+import org.spine3.samples.lobby.common.SeatTypeId;
+import org.spine3.samples.lobby.registration.contracts.SeatQuantity;
+import org.spine3.samples.lobby.registration.seat.availability.SeatQuantities;
+import org.spine3.samples.lobby.registration.seat.availability.SeatsAvailabilityId;
+import org.spine3.util.Identifiers;
+
+/**
+ * The utility class containing convenience methods for messages creation.
+ *
+ * @see Message
+ * @author Alexander Litus
+ */
+@SuppressWarnings("UtilityClass")
+public class MessageFactory {
```

Comment:
Please have utility classes specific for bounded contexts and groups of related classes.
Consider having utility class for common types, registration types, and money.